testing/file_test/test_file.cpp

```diff
@@ -93,9 +96,47 @@ struct SplitState {
   int file_index = 0;
 };
 
+static auto ExtractFilePathFromUri(llvm::StringRef uri) -> llvm::StringRef {
+  static constexpr llvm::StringRef FilePrefix = "file:/";
+  if (uri.starts_with(FilePrefix)) {
+    return uri.drop_front(FilePrefix.size());
+  }
+  return uri;
+}
+
+static auto AutoFillDidOpenParams(llvm::json::Object* params,
+                                  llvm::ArrayRef<TestFile::Split> splits)
+    -> ErrorOr<Success> {
+  auto* text_document = params->getObject("textDocument");
+  if (text_document == nullptr) {
+    return Success{};
+  }
+
+  auto attr_it = text_document->find("text");
+  if (attr_it == text_document->end() || attr_it->second != "AUTOFILL") {
+    return Success{};
+  }
+
+  auto uri = text_document->getString("uri");
+  CARBON_CHECK(uri.has_value());
```

Comment:
This seems like it will be easy for a fuzzer to hit - maybe make this return an error?

Reply:
Wasn't thinking about the fuzzer, that makes sense, will swap to an error.